app/decorators/lib/proposals/proposal_serializer_decorator.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+Decidim::Proposals::ProposalSerializer.class_eval do
+  def serialize
```

Comment:
for better maintainability maybe we can alias this method and call `original_serialize` and simply add the `author_*` fields to the hash, and return the hash